utilities/virt.py

```diff
@@ -2638,3 +2639,39 @@ def username_password_from_cloud_init(vm_volumes: list[dict[str, Any]]) -> tuple
             return _match["user"], _match["password"]
 
     return "", ""
+
+
+@contextmanager
+def enable_aaq_in_hco(client, hco_namespace, hyperconverged_resource, enable_acrq_support=False):
```

Comment:
it is not a part of virt.py

I think it should be in hco.py since we are updating the HCO here